test/Superpower.Tests/Display/PresentationTests.cs

```diff
@@ -31,7 +31,7 @@ public void ProperNameIsDisplayedWhenNonGraphicalCausesFailure()
         public void ProperNameIsDisplayedWhenNonGraphicalIsFailed()
         {
             var result=Character.EqualTo('\x2007').TryParse("a");
-            Assert.Equal("Syntax error (line 1, column 1): unexpected `a`, expected figure space.",result.ToString());
+            Assert.Equal("Syntax error (line 1, column 1): unexpected `a`, expected `figure space`.",result.ToString());
```

Comment:
I don't think there should be backticks around `figure space`, here, since it's part of the sentence rather than data extracted from the source text. The idea around the use of backticks in messages in Superpower is that they delimit literal content (so `\n` would get backticks, but `newline` would not)